db/migrate/20160527093031_limit_os_description.rb

```diff
@@ -0,0 +1,12 @@
+class LimitOsDescription < ActiveRecord::Migration
+  def up
+    Operatingsystem.where("description <> ''").each do |os|
+      os.update_attribute(:description, os.description.truncate(255))
```

Comment:
`update_attribute` triggers callbacks which might cause problems in future, could you either use [fake model](https://github.com/theforeman/foreman/blob/develop/db/migrate/20131127112625_rename_seeded_templates.rb#L39) or `.update_all` for each os?